src/redux/reducers/actionLiterals.ts

```diff
@@ -0,0 +1,7 @@
+export const changePageAction = {
+    actionType: 'page/changePage',
+    actionPayloads: {
```

Comment:
Create an enum for this. https://www.typescriptlang.org/docs/handbook/enums.html